### PR TITLE
ACN_CD_InfobloxUpdate

### PR DESCRIPTION
**Pull request by chicduong**
Fixes #

## Proposed Changes

  - Workbook Queries and Analytics Rules updated accordingly to new parser
  - Other Syslog parsers were validated and did have the same issue.

### CODE REVIEW

**Comment by preetikr** [comment]
@ashwin-patil - Since you reviewed the parser change for this, can you please check if the relevant analytics are good too. Thanks.

File: Detections/InfobloxNIOS/ExcessiveNXDOMAINDNSQueries.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ query: |
   let threshold = 200;
   InfobloxNIOS
   | where TimeGenerated >= timeframe
```
Comment (ashwin-patil) [inline Detections/InfobloxNIOS/ExcessiveNXDOMAINDNSQueries.yaml:24]: change 
`let timeframe = 1h`
and condition to 
`| where TimeGenerated > = ago(timeframe)`

File: Detections/InfobloxNIOS/ExcessiveNXDOMAINDNSQueries.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ query: |
...
-  | where LogType =~ "named" and Type =~ "client"
+  | where ProcessName =~ "named" and Log_Type =~ "client"
   | where isnotempty(ResponseCode)
   | where ResponseCode =~ "NXDOMAIN"
   | summarize count() by Client_IP, bin(TimeGenerated,15m)
```
Comment (ashwin-patil) [inline Detections/InfobloxNIOS/ExcessiveNXDOMAINDNSQueries.yaml:28]: Please join the summarized results to original table and bring in other contextual fields in the output. 
Current output does not provide any helpful information to analysts other that showing IP addresses and count. 

Also populate and map entities (IPCustomEntity, HostCustomEntity) at the end of the query.
Check example queries
- https://github.com/Azure/Azure-Sentinel/blob/master/Detections/DnsEvents/DNS_HighReverseDNSCount_detection.yaml
- https://github.com/Azure/Azure-Sentinel/blob/master/Detections/DnsEvents/DNS_HighNXDomainCount_detection.yaml

File: Detections/InfobloxNIOS/PotentialDHCPStarvationAttack.yaml [hunk 3]
```diff
@@ -21,7 +21,7 @@ query: |
   let threshold = 1000;
   InfobloxNIOS
   | where TimeGenerated >= timeframe
-  | where LogType =~ "dhcpd" and Type =~ "DHCPREQUEST"
+  | where ProcessName =~ "dhcpd" and Log_Type =~ "DHCPREQUEST"
   | summarize count() by ServerIP, bin(TimeGenerated,5m)
   | where count_ > threshold
```
Comment (ashwin-patil) [inline Detections/InfobloxNIOS/PotentialDHCPStarvationAttack.yaml:26]: Same feedback above. Bring other contextual fields with joining with original table. 
Refer examples from Github

File: Detections/InfobloxNIOS/ExcessiveNXDOMAINDNSQueries.yaml (hunk 1 above)
Comment (chicduong) [reply Detections/InfobloxNIOS/ExcessiveNXDOMAINDNSQueries.yaml]: Revisions have been made

File: Detections/InfobloxNIOS/PotentialDHCPStarvationAttack.yaml (hunk 3 above)
Comment (chicduong) [reply Detections/InfobloxNIOS/PotentialDHCPStarvationAttack.yaml:26]: join has been added

File: Detections/InfobloxNIOS/ExcessiveNXDOMAINDNSQueries.yaml (hunk 2 above)
Comment (chicduong) [reply Detections/InfobloxNIOS/ExcessiveNXDOMAINDNSQueries.yaml:28]: join has been added

**Pull request merged**